### PR TITLE
Fixes/row sizing tb headers

### PR DESCRIPTION
**Pull request by caneruguz**
## Purpose
There is a large amountof space between instructions and search filter where Treebeard instructions take place. This change includes the instructions within treebeard as a title column next to the search bar and adds proper sizing for responsiveness. 
Fixes Trello card: https://trello.com/c/dNZs1X5m

## Change
**This requires treebeard update**
Changes were made to files page, project overview page and conferences/meetings page where TB options were edited to include the instructions row

## Side effects
I checked proper sizing but responsiveness is something to keep an eye on. 
This method also uses mako for what to pass to the title html template and no direct html is being rendered. It would be useful to carefully review the mako file changes in this PR.

### CODE REVIEW

**Comment by sloria** [comment]
@caneruguz The help text wraps to 2 lines. Can you fix this show it displays on one line? 

osf___addons_testing_lol_files (Firefox 35.0)

**Comment by JeffSpies** [comment]
"To upload, drag files into a folder OR click the BUTTONs below"?

**Comment by sloria** [comment]
Or even shorter: "To upload, drag files into a folder OR click <ICON>".

**Comment by brianjgeiger** [comment]
To upload, drag files or \<icon\>

**Comment by JeffSpies** [comment]
I might click THAT icon then.

**Comment by sloria** [comment]
OK. Let's go with Jeff's suggestion for now

File: website/templates/public/pages/meeting.mako [hunk 1]
```diff
@@ -64,6 +58,15 @@
     <script type="text/javascript">
         window.contextVars = window.contextVars || {};
         window.contextVars.meetingData = ${data};
+        
+        % if meeting['active'] and meeting['info_url']:
+            window.contextVars.tbInstructions =  'withLink';
+            window.contextVars.tbInstructionsLink =  '${ meeting['info_url'] }';
```
Comment (sloria) [inline website/templates/public/pages/meeting.mako]: Why have 2 variables here? You could just have `tbInstructionsLink` and check if it is defined in `conference.js`.

File: website/static/js/fangorn.js [hunk 2]
```diff
@@ -780,9 +780,17 @@ tbOptions = {
     uploads : true,         // Turns dropzone on/off.
     columnTitles : _fangornColumnTitles,
     resolveRows : _fangornResolveRows,
+    title : function() {
+        if(window.contextVars.uploadInstruction) {
+            return m('p', [
+                m('span', 'To Upload: Drag files into a folder below OR click the ('),
```
Comment (sloria) [inline website/static/js/fangorn.js]: The parentheses around the icon are unnecessary.

**Comment by sloria** [comment]
Pass finished. :poodle:

**Comment by caneruguz** [comment]
@sloria I made the changes suggested in the last check you made. Ready for review again.

**Comment by sloria** [comment]
good looks :octopus:

**Pull request merged**